pinot-tools/src/main/java/org/apache/pinot/tools/KafkaStarterUtils.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.tools;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import org.apache.pinot.common.utils.ZkStarter;
+import org.apache.pinot.core.realtime.stream.StreamDataProvider;
+import org.apache.pinot.core.realtime.stream.StreamDataServerStartable;
+
+
+public class KafkaStarterUtils {
+  public static final int DEFAULT_BROKER_ID = 0;
+  public static final String DEFAULT_ZK_STR = ZkStarter.DEFAULT_ZK_STR + "/kafka";
+  public static int DEFAULT_KAFKA_PORT = 19092;
+  public static final String DEFAULT_KAFKA_BROKER = "localhost:" + DEFAULT_KAFKA_PORT;
+
+  public static final String PORT = "port";
+  public static final String BROKER_ID = "broker.id";
+  private static final String ZOOKEEPER_CONNECT = "zookeeper.connect";
+  private static final String LOG_DIRS = "log.dirs";
+
+  public static final String KAFKA_09_SERVER_STARTABLE_CLASS_NAME =
+      "org.apache.pinot.core.realtime.impl.kafka.server.KafkaDataServerStartable";
```

Comment:
is the expectation now that every kafka implementation should have the KafkaDataServerStartable in the path `org.apache.pinot.core.realtime.impl.kafka.server` if we want the pinot-tools/pinot-perf/pinot-integration-tests to run?

Reply:
yes, we can probably use profiles to add the right dependency? https://stackoverflow.com/questions/166895/different-dependencies-for-different-build-profiles

Reply:
I think if we want to have an integration test, then data server/provider has to be implemented. Either we can have a mock one or we directly wrapper a real implementation.